.github/workflows/ubuntu.yml

```
@@ -208,7 +208,8 @@ jobs:
           -DWarpX_LIB=ON               \
           -DWarpX_MPI=OFF              \
           -DWarpX_OPENPMD=ON           \
-          -DWarpX_PRECISION=SINGLE
+          -DWarpX_PRECISION=SINGLE     \
+          -DWarpX_openpmd_branch=dev
```

Just released a new version:
```suggestion
          -DWarpX_openpmd_branch=0.13.4
```